cardano-crypto-tests/bls12-381-test-vectors/test-vectors-bls12_381/README.md

```diff
@@ -0,0 +1,58 @@
+## Test vectors for BLS
+### 1- Test vectors for pairing properties
+- `e([a]P, Q) = e(P, [a]Q)`
+- `e([a]P, [b]Q) = e([b]P, [a]Q)`
+- `e([a]P, [b]Q) = e([a * b]P, Q)`
+- `e([a]P, Q) * e([b]P, Q) = e([a + b]P, Q)`
+
+Order of the values printed on `pairing_test_vectors`:
+- `[a]P`    
+- `[b]Q`
+- `[b]P`
+- `[a]Q`
+- `[a + b]P`
+- `[a * b]P`
+
+
+### 2- Test vectors for elliptic curve operations
```

Comment:
Missing introduction on what operations can be tested with these test vectors (addition, subtraction, scalar mult and negation)